backend/src/actions/grafana/createGrafanaDashboard.ts

```diff
@@ -39,5 +40,31 @@ export default function createGrafanaDashboardObject(
     );
   });
 
-  return dashboard;
+  try {
+    const dashboardResponse = await axios.post(
+      'http://host.docker.internal:2999/api/dashboards/db',
+      JSON.stringify(dashboard),
+      {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      }
+    );
+
+    // Descriptive error log for developers
+    if (dashboardResponse.status >= 400) {
+      console.log(
```

Comment:
NIT: Would be nice if errors logged to the `stdout` stream by using `console.error()` isntead